opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -257,19 +257,27 @@ def test_invalid_instrumentation_info(self):
             tracer1 = tracer_provider.get_tracer("")
         with self.assertLogs(level=ERROR):
             tracer2 = tracer_provider.get_tracer(None)
-        self.assertEqual(
-            tracer1.instrumentation_info, tracer2.instrumentation_info
-        )
+
```

Comment:
any reason for this test to be moved?

Reply:
I believed that having the more granular tests for the individual tracer would provide more information upon failing; then, this test just makes sure that the two tracers created are identical, which is asserted after each individual attribute is checked. I'm not sure if that's good practice, though, so I'd appreciate the feedback!